File: fibonacciModified.py
//https://www.hackerrank.com/challenges/fibonacci-modified
//python 2

arr = map(long, raw_input().split())
a = arr[0]
b = arr[1]
n = arr[2]

def fibMod(first, second, final):
    while(final>2):
        third = second*second + first
        first = second
        second = third
        final = final - 1
    print long(third)

fibMod(a,b,n)

File: fibonacciModified.js
//https://www.hackerrank.com/challenges/fibonacci-modified

function processData(input) {
    let arr = input.split(' ').map(Number);
    let a = arr[0];
    let b = arr[1];
    let n = arr[2]-2;
    
    function fibMod(first, second, final){
        let third = 0;
        while(final--){
            third = Math.pow(second, 2) + first;
            first = second;
            second = third;
        }
        return third;
    }
    console.log(fibMod(a,b,n));
} 